pkg/tftarget.go/library.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tftarget
+
+import "text/template"
+
+/*
+This comment puts the start of rego on line 10 so it's easier to do math when
+it calls out the line number.
+
+*/
+const libraryTemplateSrc = `package target
+
+matching_constraints[constraint] {
+	asset := input.review
```

Comment:
Maybe call this `resourceChange` instead of `asset` since it is not a CAI Asset

Reply:
done,

oops on the name of the folder :p .. good catch !